src/app/api/bot/longpolling/route.ts

```
@@ -0,0 +1,9 @@
+import { bot } from "@/app/telegram/bot"
```

ESLint

[error] 'bot' is defined but never used.

(@typescript-eslint/no-unused-vars)